assembl/locale/fr/LC_MESSAGES/assembl.po

```diff
@@ -306,100 +306,128 @@ msgstr ""
 " vide, n'importe qui peut s'auto-inscrire. "
 
 #: assembl/models/preferences.py:746
+#, fuzzy
+msgid "Whitelist on authentication service"
+msgstr "Type du service d'identification"
+
+#: assembl/models/preferences.py:749
+msgid "Allow white list to be applied to the authentication service chosen"
+msgstr ""
+
+#: assembl/models/preferences.py:761
+msgid "Whitelist on standard registration"
+msgstr ""
```

Comment:
Right, here you should translate the `msgid` string to French. Do this for all newly created msgids that do not have a msgstr in French yet.

Reply:
Sorry that you have to do these ones. I didn't do them and merged. So, you are getting to do a bit more work.